lib/govuk_app_config/govuk_error/configuration.rb

```diff
@@ -25,8 +25,13 @@ def should_capture=(closure)
     def ignore_excluded_exceptions_in_data_sync
       lambda { |error_or_event|
         data_sync_ignored_error = data_sync_excluded_exceptions.any? do |exception_to_ignore|
+          exception_to_ignore = Object.const_get(exception_to_ignore)
           exception_chain = Raven::Utils::ExceptionCauseChain.exception_to_array(error_or_event)
-          exception_chain.any? { |exception| exception.class.to_s == exception_to_ignore }
+          exception_chain.any? { |exception| exception.is_a?(exception_to_ignore) }
```

Comment:
I notice raven relies on the `===` method - though `is_a?` is a lot more readable and I can't imagine any reliance on subtle nuances between them. Thought I'd note it though since we got a little burnt last time I thought there wasn't a big consequence diverging from Raven 🤕

Reply:
Good to note - but I think it's worth taking the risk, as it does seem more readable given it handles subclasses etc (`===` seems quite misleading in that regard).